### ISSUE TITLE
⚠ Looking for contributors!

### ISSUE THREAD

**Issue by freetv332** [Free-TV/IPTV#211]
I'm looking for contributors to help maintain this project.

I have less time than I anticipated. The URLs change very often.
The more people help maintaining these lists the better.

Many thanks to @infid0, @KAMI911 and @NezbednikSK already!

If you have some time, if you understand what we're doing here and you want to help, don't hesitate to tag me in a comment here.

Thank you.

**Comment by KAMI911** [comment]
Hell @freetv332 , Could you enable GitHub action for this repo? I got 
actions/checkout@v2 is not allowed to be used in Free-TV/IPTV. Actions in this workflow must be: within a repository owned by Free-TV error. I would like to automate this repo as much as we can, automated tests, playlist update, and anything that makes our life easier. Thank you for your help and the project!

**Comment by infid0** [comment]
I agree with @KAMI911 , an automatic update of the playlist and a test for the streaming urls would be very convenient for this project and all contributors. :3

**Comment by NezbednikSK** [comment]
and so do i
~~a full commitment's what i'm thinking of~~

**Comment by KAMI911** [comment]
Initial set of actions are here: https://github.com/Free-TV/IPTV/actions , but I cannot run them.

**Comment by freetv332** [comment]
@KAMI911 and now?

**Comment by freetv332** [comment]
Actions run well now.

**Comment by infid0** [comment]
Wow, the playlist updated in real time is absolutely awesome! Good job guys! :D

**Comment by Kinggmahdi** [comment]
pls add iran channel

**Comment by KAMI911** [comment]
Issue regarding to Iran channels: #239

**Comment by Nsomnia** [comment]
I only do rips, merges, and dedupes of artiptv.net

**Comment by Rameshharesh** [comment]
How get free tv

**Comment by adilaxmdv** [comment]
I want join your team

**Comment by adilaxmdv** [comment]
@freetv332

**Comment by mu1664al-s** [comment]
Hi there. I would like to help. I can contribute with TV channels for Turkey, Iraq and Sweden.

**Comment by diamondjohndroid** [comment]
I cant help, BUT I have a list of IP TV Channels based in AUS I didn't find in your HUGE list. Every time I look they change :(

A lot of these are Free To Air Standard DVB TV Channels in AUS, that I use for when reception is bad.
[AUS - Sydney - IPTV.zip](https://github.com/Free-TV/IPTV/files/13405480/AUS.-.Sydney.-.IPTV.zip)

**Comment by dp247** [comment]
@freetv332 I'm more than happy to approve some of the PRs - I'm used to contributing to these kinds of projects and I've already done some here 😄

**Comment by voxel79** [comment]
> AUS - Sydney - IPTV.zip

are these geoblocked btw?
@diamondjohndroid

**Comment by Avniboy** [comment]
I can be of a good help too
thnx

**Comment by Avniboy** [comment]
@freetv332 add channels to kosovo - i sent via issue, those are my channels so dont worry of anything else
I also can be of a good help on tv, contact me

**Comment by Chrisoo123** [comment]
hi there i have an m3u playlist for lebanon. an i contribute?

**Comment by senthilkumar910** [comment]
H

**Comment by KAMI911** [comment]
Are you all still interested? Please notify me. I try to handle pull request in priority. Also can you help to make decision regarding the remaining issues and PRs?

**Comment by dp247** [comment]
> Are you all still interested? Please notify me. I try to handle pull request in priority. Also can you help to make decision regarding the remaining issues and PRs?

Sure thing. I've been keeping an up-to-date fork so I'll happily help out still.

**Comment by KAMI911** [comment]
Could you help out with a pull request, that would be awesome at first. Do you keep the rules of this list described in the README?

**Comment by Chrisoo123** [comment]
hi there, here is the m3u file zipped. 
[lb.m3u.zip](https://github.com/user-attachments/files/17124037/lb.m3u.zip)

**Comment by olegcherev** [comment]
Hi @freetv332 
do you think the implementation of gamification elements for the contribution process may help to engage the community? 

I made a GitHub app that implements elements like XP for commits/PRs/issues, Levels, Leaderboard, Challenges, Badges. This may work for community, especially first-time contributors to contribute more with no efforts from maintainer.

There's a [Live Demo](https://demo.devactivity.com/achievements/?activeTab=achievements&organizationId=1) (no auth required) and [Public Dashboard Demo](https://demo.devactivity.com/public/?organizationLogin=demo-organization&organizationId=1)

**Comment by erictonto** [comment]
Got it.


Sent from Yahoo Mail for iPhone


On Sunday, October 20, 2024, 12:35 PM, Oleg ***@***.***> wrote:




Hi @freetv332
do you think the implementation of gamification elements for the contribution process may help to engage the community?

I made a GitHub app that implements elements like XP for commits/PRs/issues, Levels, Leaderboard, Challenges, Badges. This may work for community, especially first-time contributors to contribute more with no efforts from maintainer.

There's a Live Demo (no auth required) and Public Dashboard Demo

**Comment by VlastikYoutubeKo** [comment]
@freetv332 I'm interested in helping you with this.
Although I don't have too much free time, I could make some to check PRs etc.

**Comment by frostoutside** [comment]
Damn this is really dead.

**Comment by dp247** [comment]
> Damn this is really dead.

I still keep my fork semi up-to-date, though no guarantee a lot of the country channels are working 😅

**Comment by KAMI911** [comment]
If you create a pull request, I will try to merge them. Sorry for the slow process, I have many things around.

**Comment by PyProInitiate** [comment]
@freetv332

**Comment by samethernet** [comment]
@freetv332 , I have recently found this repository and am really happy with what i'm seeing hence my own pull request to update few incorrect and broken streams. I'm more than happy to contribute in the aspect of testing streams that come through pull requests and merging them into the repository. I can be somewhat active like once a week to follow up on this. Added benifit I'm from Australia so might even be able to test and add some AU geolocked channels as I become aware of them.

**Comment by tomaszmarta13-dotcom** [comment]
> Szukam osób chętnych do pomocy w utrzymaniu tego projektu.
> 
> Mam mniej czasu, niż się spodziewałem. Adresy URL zmieniają się bardzo często. Im więcej osób pomoże w utrzymaniu tych list, tym lepiej.
> 
> Wielkie dzięki[@infid0](https://github.com/infid0),[@KAMI911](https://github.com/KAMI911)I[@NezbednikSK](https://github.com/NezbednikSK)już!
> 
> Jeśli masz trochę czasu, jeśli rozumiesz, co tu robimy i chcesz pomóc, nie wahaj się oznaczyć mnie w komentarzu.
> 
> Dziękuję.

**Comment by Avniboy** [comment]
Srdjan can u write in private to me pls
***@***.***

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Dadu042** [comment]
Hello, as a user of the software Hypnotix, I discover this github. I see that many pull requests are waiting. I wish to help to review some, mainly for france streams.

**Comment by 2026TVPlus** [comment]
hello can i be a contributor? they banned me from iptv org and i know a ton about iptv etc. so i'm looking for something to do

<img width="1919" height="1079" alt="Image" src="https://github.com/user-attachments/assets/91a4a74e-cd15-4679-8187-01df8fb77cd8" />

**Comment by KAMI911** [comment]
You can help of course, free to check issues, the provide pull requests for md files, @2026TVPlus

**Comment by KAMI911** [comment]
@Dadu042 , always can help with check links and fix them with a pull request.

**Comment by AlbertAnimates** [comment]
### Heya!
- I did my own IPTV service (check my account)
- I want to contribute

**Comment by KAMI911** [comment]
That would be awesome, @AlbertAnimates. Could you create PR against the master for *.md files regarding to invalid, non working streams?

**Comment by KAMI911** [comment]
Any others to help?

**Comment by KAMI911** [comment]
@freetv332 , @AlbertAnimates , @2026TVPlus , @BaleshSrle , @manoharpeddinti-embedded-engineer , @samethernet @VlastikYoutubeKo , @dp247 , @Chrisoo123 , @infid0 , @NezbednikSK , @mu1664al-s , @diamondjohndroid it is time to ask your help again, if you can help. I cleaned up the PRs and Issues, fixed automation, and the IPTV checker is publising the faulty channels (but those could be false for wring timing, for geoblocking, or network issues. More than 400 channels are fixed, and autotested. I hope the health of this project is recovered.

**Comment by dp247** [comment]
You the man @KAMI911!

I've been maintaining the list of english news channels (specifically non-YT because my software only works with direct links), so I'll sync my fork and merge those in 😄
